eng/pipelines/azure-pipelines.yml

```diff
@@ -112,11 +112,43 @@ extends:
 
     stages:
 
+    - stage: build_sign_native
+      displayName: Build+Sign native packages
+
+      jobs:
+      - template: /eng/pipelines/templates/build_sign_native.yml@self
+        parameters:
+          agentOs: macos
+          targetRidsForSameOS:
+            - osx-arm64
+            - osx-x64
+          codeSign: true
+          teamName: $(_TeamName)
+          extraBuildArgs: >-
+            /p:Configuration=$(_BuildConfig)
+            $(_SignArgs)
+            $(_OfficialBuildIdArgs)
+
+      - template: /eng/pipelines/templates/build_sign_native.yml@self
+        parameters:
+          agentOs: linux
+          targetRidsForSameOS:
+            - linux-x64
```

Comment:
Isn't this missing arm64 and musl?

Reply:
We don't AOT those yet, so they are being built in the main Windows job instead of having separate agents to produce self-contained executables.

Reply:
Can you help me understand why? Why don't we aot those yet? I thought all we needed was to match the OS, so I would have expected that this same queue (the linux-x64 one) to be able to aot for those other two.

Reply:
We need to set the machine up for cross compliation.

See [Publish NativeAOT on Ubuntu doesn't work with -r linux-musl-x64 (dotnet/runtime#92294)](https://github.com/dotnet/runtime/issues/92294)

and https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/cross-compile#linux

Reply:
The scripts PR has a README where we mention this - https://github.com/dotnet/aspire/pull/10254/files#diff-e2e2432bd7a67b477d58cdba86311ddd7d77d117ba1805bf70545131e380e0dd .

Reply:
.. and in the PR description.